.travis.yml

```diff
@@ -9,8 +9,8 @@ env:
   - TEST_COMMAND=test EXTRA_FLAGS='--no-default-features' FEATURES='std u32_backend'
   # Tests the u64 backend
   - TEST_COMMAND=test EXTRA_FLAGS='--no-default-features' FEATURES='std u64_backend'
-  # Tests the avx2 backend
-  - TEST_COMMAND=test EXTRA_FLAGS='--no-default-features' FEATURES='std avx2_backend'
+  # Tests the simd backend
+  - TEST_COMMAND=test EXTRA_FLAGS='--no-default-features' FEATURES='std simd_backend'
```

Comment:
so the avx2 backend is deprecated my the simd one or we should support both?

Reply:
It's a refactoring to avoid conflicts on some AMD processors:
https://github.com/dalek-cryptography/curve25519-dalek/pull/334
We've never used either directly.